frame/composable-traits/src/financial_nft.rs

```diff
@@ -30,6 +37,11 @@ pub trait FinancialNftProvider<AccountId>: Create<AccountId> + Mutate<AccountId>
 		version: &K,
 		nft_data: &V,
 	) -> Result<Self::InstanceId, DispatchError>;
+
+	fn split(
+		instance: &Self::InstanceId,
+		overrides: BiBoundedVec<BTreeMap<AttributeKey, AttributeValue>, 1, 16>,
```

Comment:
What does overrides do?

Reply:
will move docs from impt to trait in next PR